kahuna/public/stylesheets/main.css

```diff
@@ -418,6 +440,61 @@ textarea.ng-invalid {
     display: inline-block;
 }
 
+.search__filters li {
+    display: inline;   
+}
+
+.search__filters-container {
+    position: relative;
+    display: inline-block;
+}
+
+.search__filters-toggle {
+    display: none;
+}
+
+@media screen and (max-width: 614px) {
+    .search__advanced-toggle {
+        display: none;
+    }
+}
+@media screen and (max-width: 700px) {
+    input.search-query__input {
+        width: 170px
+    }
+
+}
+@media screen and (max-width: 1054px) {
```

Comment:
The crop page needs updating too?

![screen shot 2015-07-15 at 10 52 19](https://cloud.githubusercontent.com/assets/836140/8695854/e4deebf0-2adf-11e5-8213-10ee9778b05c.png)